doc/api/v8.md

```diff
@@ -271,6 +271,78 @@ buffers and external strings.
 }
 ```
 
+## `v8.getCppHeapStatistics(detailLevel)`
+
+This method returns [CppHeap][] statistics regarding memory consumption and
+utilization. See V8 [`CollectStatistics()`][] API. It returns a
+[`HeapStatistics`][] object. You can learn more about the properties in
+the [V8 documentation](https://v8docs.nodesource.com/node-22.4/df/d2f/structcppgc_1_1_heap_statistics.html).
```

Comment:
It is worth pointing out that this object structure may change from one v8 version to the next, even if unlikely.